libraries/tools/kotlin-gradle-plugin/src/test/kotlin/org/jetbrains/kotlin/gradle/internal/kapt/incremental/ClasspathAnalyzerTest.kt

```diff
@@ -172,7 +167,9 @@ class ClasspathAnalyzerTest {
 }
 
 class StructureTransformTestAction(val input: File) : StructureTransformAction() {
-    override val inputArtifact: Provider<FileSystemLocation> = FakeGradleProvider(FakeGradleRegularFile(input))
+    private val project = ProjectBuilder.builder().build()
```

Comment:
Can you please specify `withProjectDir​(File dir)` option to make sure we remove the project dir?

Reply:
Done